indico/modules/categories/operations.py

```diff
@@ -20,6 +20,24 @@
 
 from indico.core.db import db
 from indico.modules.categories import logger
+from indico.modules.categories.models.categories import Category
+
+
+def create_category(parent, data):
+    category = Category(parent=parent)
+    data.setdefault('default_event_themes', parent.default_event_themes)
+    data.setdefault('timezone', parent.timezone)
```

Comment:
IMHO it makes more sense to use the value from `data` if it's present for some reason - operations are supposed to be somewhat generic (i.e. the fact that only title/description are provided in the form is outside the scope of the operation)